apps/admin-x-activitypub/src/components/feed/ArticleModal.tsx

```diff
@@ -448,7 +448,7 @@ const ArticleModal: React.FC<ArticleModalProps> = ({
         // Add the new reply to the thread
         activity.object.authored = true;
         activity.id = activity.object.id;
-        addToThread(activity);
+        addToThread(activity, true);
```

Comment:
`addToThread` is only ever used here - Maybe adding to the top of the thread should always be the default behaviour rather than us having conditional branching?

Reply:
Good point!